packages/wdio-cucumber-framework/src/wdioEventTranslator.js

```diff
@@ -5,10 +5,28 @@ import * as path from 'path'
 import logger from '@wdio/logger'
 
 const SETTLE_TIMEOUT = 5000
-const log = logger('wdio-cucumber-framework')
+const log = logger('wdio-cucumber-framework:CucumberWdioEventTranslator')
```

Comment:
This is pretty cumbersome. While i understand wanting to know exactly where its logging from, that eats up considerable real estate.

Reply:
Makes sense. For now that's super useful while I'm still working out bugs, but I'll make sure it's out at the end!